### DISCUSSION TITLE
Code Compilation

### DISCUSSION THREAD

**Discussion by wWL2771140158** [xcompact3d/Incompact3d#195] [Q&A] [answered]
Hello, is the installation and compilation of incompact3d related to the processor? Can only use an Intel processor? Or can AMD processors also work?

**Comment by pbartholomew08** [comment, marked as answer]
Hi, all you should need to get started is an MPI Fortran compiler for your machine. Incompact3d has been run on Intel, AMD, arm and IBM machines.

Hope that helps!
